vector/src/main/java/im/vector/app/features/spaces/explore/SpaceDirectoryViewModel.kt

```diff
@@ -40,7 +40,7 @@ import timber.log.Timber
 
 class SpaceDirectoryViewModel @AssistedInject constructor(
         @Assisted initialState: SpaceDirectoryState,
-        private val session: Session
+        val session: Session
```

Comment:
why this change? If it is necessary, I do not like it :/

Reply:
yeah i took a shortcut as it's temporary, but will add action/events

Reply:
done